crossbeam-deque/src/deque.rs

```diff
@@ -1603,6 +1715,41 @@ impl<T> Injector<T> {
     /// assert_eq!(w.pop(), Some(2));
     /// ```
     pub fn steal_batch_and_pop(&self, dest: &Worker<T>) -> Steal<T> {
+        self.steal_batch_with_limit_and_pop(dest, MAX_BATCH)
```

Comment:
I'm not sure if the default of `Injector::steal_batch_and_pop` needs to be changed from `MAX_BATCH + 1`.

Reply:
Yes, I also thought about it, as the number of elements in each call is not guaranteed by default and the caller should not depend on it, use `MAX_BATCH` or `MAX_BATCH + 1` should both ok here, but I prefer use `MAX_BATCH` to be the same with Stealer, `MAX_BATCH + 1` seems a bit weird to me.

Reply:
It may have been a performance-related tweak (https://github.com/crossbeam-rs/crossbeam/commit/af8ce60f8b32104e489859b60e3d9ae7e3de59a8). So, I tend to keep the previous one for now, but it might make sense to have a TODO comment indicating that it may need to be revisited in the future.

Reply:
done